control/controlknobs/controlknobs.go

```diff
@@ -76,6 +76,10 @@ type Knobs struct {
 	// AppCStoreRoutes is whether the node should store RouteInfo to StateStore
 	// if it's an app connector.
 	AppCStoreRoutes atomic.Bool
+
+	// UserDialUseRoutes is whether tsdial.Dialer.UserDial should use routes to determine
+	// how to dial the destination address.
+	UserDialUseRoutes atomic.Bool
```

Comment:
this is temporary, right? can you add a URL to a tracking bug in here? so we know when we can remove this, or what's tracking removing it, or more context on why this exists.

(in retrospect all the above should probably have URLs too. I'll do that later.)

Reply:
Done.